crm_construction_architect/README.rst

```diff
@@ -0,0 +1,37 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Construction Architect CRM
+===========
+
+This module allows you to assign an architect to a lead or opportunity.
+
+Installation
+============
+
+To install this module, you need to:
+
+* have construction_architect_base available
```

Comment:
wrong module name, and I'd remove the paragraph altogether as this is handled by odoo automatically.